### PR TITLE
Remove PR field and fix other minor changes

### PR DESCRIPTION
**Pull request by nis130**

### CODE REVIEW

File: src/components/MotivationForm.tsx [hunk 1]
```diff
@@ -68,7 +64,7 @@ const MotivationForm = ({
 
     POST({ ...formData }).then((res) => {
       console.log(res);
-      if (res.message === "Yayay") {
```
Comment (nis130) [inline src/components/MotivationForm.tsx:71]: Hey @pettiboy, why is this change required?

Comment (pettiboy) [reply src/components/MotivationForm.tsx:71]: `api/post.ts` is directly returning string instead of `request`.
